prijateli_tree/app/routers/games.py

```diff
@@ -160,6 +160,9 @@ def get_previous_answers(
     # Get current round
     current_round = get_current_round(game_id, db)
 
+    # Check if names are hidden or not
+    names_hidden = game.game_type.names_hidden
+
```

Comment:
Can you define this around when it is used? E.g. line 191.  Well have to think less when we read the code in the future.

Reply:
I agree with Michael. So follow his advice for both of my issues.